sale_commercial_partner/readme/CONFIGURE.md

```diff
@@ -0,0 +1,7 @@
+- Go to **Sales → Configuration → Settings**.
+- Enable **Filter Invoice Address by Customer’s Commercial Partner**.  
+  This restricts the invoice address selection to partners whose commercial
+  partner matches the sales order’s customer.
+- Enable **Filter Shipping Address by Customer’s Commercial Partner**.  
+  This restricts the shipping address selection to partners whose commercial
+  partner matches the sales order’s customer.
```

Comment:
Please include a note that these options should not be used with sale_partner_address_restrict.